xml/ay_bigfile.xml

```diff
@@ -4489,6 +4489,36 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
     <remark role="fixme">Add a short description</remark>
    </para>
 
+   <sect2 xml:id="Software.Selections.base_product">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You have to
+     choose which product you wish to install. You can do it explicitly using.
+     <literal>product</literal> option
+     <example>
+       <title>Explicit product selection</title>
+       <para>
+         &lt;software&gt;&lt;products&gt;&lt;product&gt;SLED15&lt;/product&gt;&lt;/products&gt;&lt;/software&gt;
+       </para>
+     </example>
+    </para>
+    <para>
+      In some special cases it can happen that the media contains just one product. In
+      such case you don't need to select a product explicitly as described above.
+      AutoYaST will select the only available product automatically.
+    </para>
+    <para>
+      This paragraph descibes deprecated behavior which is available to provide
+      minimal backward compatibility for old profiles.
+      Appart from the explicit product selection, AutoYaST2 implements a heuristic for
```

Comment:
Typo: Appart -> Apart

Also, I think, we have stopped adding the "2" to the name of (Auto)YaST -- I guess it is best to use the entity &ay; instead.